Gruntfile.js

```diff
@@ -0,0 +1,45 @@
+module.exports = function( grunt ) {
+
+	'use strict';
+	var banner = '/**\n * <%= pkg.homepage %>\n * Copyright (c) <%= grunt.template.today("yyyy") %>\n * This file is generated automatically. Do not edit.\n */\n';
+	// Project configuration
+	grunt.initConfig( {
```

Comment:
sure! I would say do it via webpack though. regardless, outside the scope of this PR